File: src/Chapter-4-C99/Questions/Question2.c
//Extend Exercise 1 to include a three digit number
#include <stdio.h>

int main(void)
{
    int first, second, third;
    printf("Enter a three-digit number: ");
    scanf("%1d%1d%1d", &first, &second, &third);
    printf("The reversal is: %1d%1d%1d", third, second, first);
    return 0;
}

File: src/Chapter-4-Rust/Questions/Question2.rs
//Extend Exercise 1 to include a three digit number
//Compute a Universal Product Code check digit
#[macro_use] extern crate text_io;
//process library to pause console
use std::process::Command;
//Need this to use the print!
use std::io::{self, Write};

fn main()
{
	let num : i32;
	print!("Enter a three-digit number: ");
	io::stdout().flush().unwrap();
	
	scan!("{}", num);
	print!("The reversal is: {}{}{}\n", num%10, (num/10)%10, num/100);	
	io::stdout().flush().unwrap();
	let _ = Command::new("cmd.exe").arg("/c").arg("pause").status();
}
